src/main/webapp/js/courseSelector.js

```diff
@@ -140,6 +154,22 @@ export const CollegePlanner = (() => {
     liElement.appendChild(deleteButtonElement);
     return liElement;
   }
+
+/**
+ * Creates an alert with the specified message in the container
+ * @param {string} message The message string you want to be displayed
+ * @param {string} type type of alert you want to display (primary, secondary,
+ *     success, warning, danger)
+ * @param {Object} container the container you want to display the alert in
+ */
+function createAlert(message, type, container) {
```

Comment:
In theory, we should probably put this function in another file entirely (we should have access to it even if courseSelector isn't here), but this is fine for now. We can move it later if needed.